BeatSaberHTTPStatus/BeatSaberHTTPStatusPlugin.csproj

```diff
@@ -155,15 +162,13 @@
       <Using Namespace="System" />
       <Using Namespace="System.IO" />
       <Using Namespace="System.Text.RegularExpressions" />
-      <Code Type="Fragment" Language="cs">
-        <![CDATA[
+      <Code Type="Fragment" Language="cs"><![CDATA[
           Log.LogMessage(string.Format("Replacing {0} in {1} to {2}", MatchExpression, InputFilename, ReplacementText));
           File.WriteAllText(
             OutputFilename,
             File.ReadAllText(InputFilename).Replace(MatchExpression, ReplacementText)
           );
-        ]]>
-      </Code>
+        ]]></Code>
```

Comment:
Either keep the change and fix the indent of the code in this block, or get rid of the change entirely.